base_layer/core/src/blockheader.rs

```diff
@@ -23,6 +23,18 @@
 // Portions of this file were originally copyrighted (c) 2018 The Grin Developers, issued under the Apache License,
 // Version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0.
 
+//! Blockchain state
```

Comment:
```suggestion
//! Blockchain state
//!
//! For [technical reasons](https://www.tari.com/2019/07/08/tari-protocol-discussion-42.html), the commitment in
//! block headers commits to the entire TXO set, rather than just UTXOs using a merkle mountain range.
```